CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,22 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
```

Comment:
I'd like this not to mean that one cannot post or reshare NSFW material, which is not imho a sign of disrespect to anyone. I'm not sure how to phrase that, so it should be maybe up to common sense to only react to cases where someone posts sexualized imagery that is somehow insulting.